@navikt/core/react/src/layout/base/BasePrimitive.tsx

```diff
@@ -142,12 +182,47 @@ export interface BasePrimitiveProps {
   flexGrow?: ResponsiveProp<string>;
 }
 
+export const BASE_PRIMITIVE_ALLOWED_PROPS: (keyof PrimtiveBaseProps)[] = [
+  "className",
+  "padding",
+  "paddingInline",
+  "paddingBlock",
+  "margin",
+  "marginInline",
+  "marginBlock",
+  "width",
+  "minWidth",
+  "maxWidth",
+  "height",
+  "minHeight",
+  "maxHeight",
+  "position",
+  "inset",
+  "top",
+  "right",
+  "bottom",
+  "left",
+  "overflow",
+  "overflowX",
+  "overflowY",
+  "flexBasis",
+  "flexGrow",
+  "flexShrink",
+];
+
+export interface BasePrimitiveProps extends PrimtiveBaseProps {
```

Comment:
`BasePrimitiveProps` og `PrimitiveBaseProps` 😵 (tenker litt på bedre naming). Og ref lage en const fra typen, det er kanskje enklere å gå andre veien her? 🤔 (type fra const?)

Reply:
og vi _må_ vel ha en const / ekte verdi pga det brukes i kall til `omit(...)` 🤔

Reply:
Jepp trenger en "ekte" verdi for bruk i omit. Tar gjerne mot forslag på naming!